feat: add IaC documentation to the README (#380)
snyk/broker: README.md

       snyk/broker:bitbucket-server
```

#### Infrastructure as Code (IaC)

> By default, the Broker does not have access to IaC configuration files. To grant the Broker access to IaC files in your repository you will need to edit your `accept.json` and add to it additional IaC specific rules.

Reviewer: ```suggestion
By default, some file types used by Infrastructure-as-Code (IaC) files are not enabled. To grant the Broker access to IaC files in your repository, such as Terraform for example, edit your `accept.json` and add the relevant IaC specific rules.
```

> For example, if you are using GitHub and you would like to give the Broker access to your TF files you will need to add the following rules to your `accept.json`:

Reviewer: ```suggestion
For example, if you are using GitHub and you would like to give the Broker access to your Terraform files you should add the following rules to your `accept.json`:
```

```console
{

>   "//": "used to determine Terraform issues",

Reviewer: nit:
```suggestion
  "//": "used to enable scanning of Terraform files",
```

  "method": "GET",
  "path": "/repos/:name/:repo/contents/:path*/*.tf",
  "origin": "https://${GITHUB_TOKEN}@${GITHUB_API}"
},
{

>   "//": "used to determine Terraform issues",

Reviewer: same comment as the above.
